app/src/hooks/useMarkets.tsx

```diff
@@ -1,6 +1,7 @@
 import { useQuery } from '@apollo/react-hooks'
 import React from 'react'
 
+import { useConnectedWeb3Context } from '../contexts/connectedWeb3'
```

Comment:
You can import it directly from the contexts folder since you created index file which exports it...Like this
![Screenshot 2021-07-16 at 12 06 00](https://user-images.githubusercontent.com/33226956/125931389-ce9cd827-43ee-4288-864d-2c8d0b1e3d46.png)